src/AdvancedPS.jl

```diff
@@ -0,0 +1,66 @@
+### Attention this is a development package! It wount run.
+
+
+module AdvancedPS
+
+
+
+        const DEBUG = Bool(parse(Int, get(ENV, "DEBUG_APS", "0")))
+        using Libtask
+        using StatsFuns: logsumexp, softmax!
+        import Base.copy
+
+        abstract type AbstractTaskInfo end
+        abstract type AbstractParticleContainer end
+        abstract type AbstractTrace end
+
+
+        include("tasks.jl")
+
+        export  ParticleContainer,
```

Comment:
All of these export blocks should be consolidated into one. It's a little hard to track what's being exported where.